package.json

```diff
@@ -64,8 +64,8 @@
     "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.803.25",
```

Comment:
Comment for line 6 -- if you update to alpha.4, the compiler should catch any usages of the endpoints you have removed (although you probably already caught them all)

Reply:
Worth a shot